.github/workflows/sync_typeshed.yml

```
@@ -5,20 +5,20 @@ on:
   schedule:
     - cron: "0 0 1,15 * *"
 
```

Instead of removing, perhaps drop the global permissions entirely?
```suggestion

permissions: {}
```